docker-compose.yml

```diff
@@ -1,11 +1,30 @@
-version: '2'
+version: '3.7'
 
 services:
-  db:
-    image: postgres:11
+  {{name}}:
```

Comment:
Since we're pulling in `web.Dockerfile`, would it make sense to call this service `web`?

Reply:
On second thought, maybe `web.Dockerfile` should be renamed to `app.Dockerfile` or just `Dockerfile`. Seems kind of weird to use the word "web" when many Vapor projects are APIs.

Reply:
> maybe web.Dockerfile should be renamed to app.Dockerfile or just Dockerfile

I intuitively thought of it like this anyway since nothing in the Dockerfile seems particularly tailored for one time of service than another. 

I can rename before we merge this PR.

Reply:
Yeah let's call it just `Dockerfile` so that `docker build .` works without needing the `-f` flag. Then using `{{ name }}` or `"app"` seem like solid choices for the service name.

Reply:
renamed to `Dockerfile`, updated `docker-compose` (no longer needed to specify a dockerfile name), updated manifest to reflect new name. Anywhere else need updating for new filename?